src/OpenCL.cpp

```diff
@@ -716,17 +720,21 @@ void OpenCL::initialize(const int channels) {
     auto found_device = false;
     auto id = 0;
 
-    myprintf("Detected %d OpenCL platforms.\n", platforms.size());
+    if(!silent) {
```

Comment:
nit: space after if for consistency with existing code (same thing in many places in this patch)